### DISCUSSION TITLE
rust-course/推荐初学者阅读的项目在哪里？

### DISCUSSION THREAD

**Discussion by mysteryven** [sunface/rust-course#1110] [内容建议]
之前阅读的时候，sunface 有一篇文章是关于推荐初学者适合看的仓库的。还记得里面还包含了 command-line Rust、用 Rust 写编辑器的文章。

当时忘记收藏了，现在找不到了，有同学知道在什么地方吗？

**Comment by sunface** [comment]
hi，我也不知道你说的哪个，大概给出两个参考:
1. https://practice.rs/elegant-code-base.html
2. https://rusty.rs/about.html
